components/sections/home-page/GitHubMock.tsx

```diff
@@ -87,7 +88,5 @@ const GitHubMock: FC<GitHubMockProps> = ({ moreHeading, topFeature }) => {
         </div>
       </div>
     </SectionWrapper>
-  )
```

Comment:
This parenthesis has been removed, this is probably the cause of your syntax error.

Reply:
nice catch @shiftyp! I missed it when I looked at it. @CBID2 you can leave these changes and apply the same in `components/sections/teams/GitHubMock.tsx`.